antora-ui-camel/src/css/frontpage.css

```diff
@@ -232,7 +232,7 @@ section.frontpage h1 {
   }
 
   header.frontpage svg {
-    display: none;
+    display: visible;
```

Comment:
Perhaps removing the whole rule would serve the same purpose?

Reply:
Ok.Sure.Thats a great Idea.I was too thinking about the abnormal font-size of the <h1> tag content as soon as the screen was reduced from 1023px.
I will be changing the font-size and will also increase the opacity of the shadow of the <h1> tag's content.